components/brave_wallet_ui/components/desktop/lock-screen/index.tsx

```diff
@@ -99,6 +102,39 @@ export const LockScreen = () => {
 
   const isAndroid = loadTimeData.getBoolean('isAndroid') || false
 
+  if (isAndroid) {
+    // Computed
+    let lastPress = 0
+
+    // Methods
+    const onDoubleTap = () => {
+      const time = new Date().getTime()
+      const delta = time - lastPress
+      if (delta < 400) {
+        getWalletPageApiProxy().pageHandler.unlockWalletUI()
```

Comment:
Maybe use the generic proxy getter?
```suggestion
        getWalletApiProxy().pageHandler?.unlockWalletUI()
```

Reply:
Updated!